generators/Output/Templates/_AssertNotEqual.liquid

```diff
@@ -0,0 +1 @@
+﻿Assert.NotEqual({{ ExpectedParameter }}, {{ TestedValue }});
```

Comment:
Doesn't look like this file is used at all

Reply:
Yes thanks!